ADotNet/Models/Pipelines/GithubPipelines/DotNets/WorkflowDispatchEvent.cs

```diff
@@ -0,0 +1,11 @@
+﻿// ---------------------------------------------------------------------------
+// Copyright (c) Hassan Habib & Shri Humrudha Jagathisun All rights reserved.
+// Licensed under the MIT License.
+// See License.txt in the project root for license information.
+// ---------------------------------------------------------------------------
+
+namespace ADotNet.Models.Pipelines.GithubPipelines.DotNets
+{
+    public class WorkflowDispatchEvent
+    { }
```

Comment:
This seems incomplete, I was expecting properties

```suggestion
    { 
        [YamlMember(Order = 0, DefaultValuesHandling = DefaultValuesHandling.OmitDefaults)]  
        public WorkflowDispatchInput Environment { get; set; }  

        [YamlMember(Order = 1, DefaultValuesHandling = DefaultValuesHandling.OmitDefaults)]  
        public WorkflowDispatchInput Version { get; set; }  

        [YamlMember(Order = 2, DefaultValuesHandling = DefaultValuesHandling.OmitDefaults)]  
        public WorkflowDispatchInput DryRun { get; set; }  
    }
```

with

```cs
public class WorkflowDispatchInput
{
    [YamlMember(Order = 0]
    public string Description { get; set; }

    [YamlMember(Order = 1]
    public bool Required { get; set; }

    [YamlMember(Order = 2]
    public string Default { get; set; }
}
```